msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/mdm/TestCase833526.java

```diff
@@ -51,6 +52,7 @@
 // https://identitydivision.visualstudio.com/DevEx/_workitems/edit/833526
 @SupportedBrokers(brokers = BrokerCompanyPortal.class)
 @RetryOnFailure(retryCount = 2)
+@Ignore("We can't automatically delete devices at the moment, so ignoring MDM_CA Enrollment test")
```

Comment:
What is wrong at the moment?

Reply:
We get this screen without the rest of the devices appearing:

![image](https://user-images.githubusercontent.com/88730756/235547553-07836ebf-7cf5-4e14-b9c2-b5301bb95617.png)

Reply:
Can we delete using the lab api?

Reply:
We're seeing the list again after Ryan did a mass deletion on his side. I'm checking with him about the deletedevice api